### PR TITLE
Simplify the automation to run the E2E tests

### PR DESCRIPTION
**Pull request by iblancasa**
Signed-off-by: Israel Blancas <iblancasa@gmail.com>

## Which problem is this PR solving?
- Simplify the E2E test automation

## Short description of the changes
- Remove some scriptts
- Move logic related to the creation and configuration of the cluster to a single place
- Other small improvements

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -407,17 +407,28 @@ ifeq ($(USE_KIND_CLUSTER),true)
...
-# When the Kind cluster is not created by Kuttl, the
-# kindContainers parameter from kuttl-tests.yaml has not effect so, it is needed to load the
-# container images here.
+# When the Kind cluster is not created by Kuttl, the kindContainers parameter
+# from kuttl-tests.yaml has not effect so, it is needed to load the container
+# images here.
 	$(VECHO)$(KIND) create cluster --config $(KIND_CONFIG) 2>&1 | grep -v "already exists" || true
+# Install metrics-server for HPA
+	$(ECHO)"Installing the metrics-server in the kind cluster"
+	$(VECHO)kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/download/v0.6.1/components.yaml
```
Comment (frzifus) [inline Makefile:416]: In terms of consistency, does it make sense to move that link/version up like other external links?

https://github.com/jaegertracing/jaeger-operator/blob/2ccf2d4a4ab799ba07a4c408bde8a2caad3d16f8/Makefile#L35-L46

File: Makefile [hunk 2]
```diff
@@ -407,17 +407,28 @@ ifeq ($(USE_KIND_CLUSTER),true)
...
+	$(VECHO)kubectl patch deployment -n kube-system metrics-server --type "json" -p '[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": --kubelet-insecure-tls}]'
+# Install the ingress-controller
+	$(ECHO)"Installing the Ingress controller in the kind cluster"
 	$(VECHO)kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.1/deploy/static/provider/kind/deploy.yaml
```
Comment (frzifus) [inline Makefile:420]: same here

**Review by frzifus — COMMENTED** [review]
hey, only one question

File: Makefile (hunk 1 above)
Comment (iblancasa) [reply Makefile]: Good idea!

**Review by frzifus — APPROVED** [review]

**Pull request merged**